File: ProyectoWebFront/src/app/dto/procesoDto.ts
// Si tienes un enum real del backend, úsalo:
export type ProcessStatus = 'DRAFT' | 'PUBLISHED' | 'INACTIVE' ;

export class ProcesoDto {
  constructor(
  public id?: number,
  public name?: string,
  public description?: string,
  public category?: string,
  public status?: ProcessStatus,
  public organizationId?: number,
  public activityIds?: number[],
  public archIds?: number[],
  public gatewayIds?: number[]){

  }
}

File: ProyectoWeb-main/src/main/java/com/example/proyectoweb/Dto/ProcesoDto.java
// DTOs Proceso/Historial/Rol
package com.example.proyectoweb.Dto;

import com.example.proyectoweb.common.ProcessStatus;
import lombok.*;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProcesoDto {
    private Long id;
    private String name;
    private String description;
    private String category;
    private ProcessStatus status;
    private Long organizationId;
    private List<Long> activityIds;
    private List<Long> archIds;
    private List<Long> gatewayIds;
}
